src/unittest/Makefile.am

```diff
@@ -0,0 +1,19 @@
+## This file lists tests that do not require mock objects to run.
+## To link your test executable against tincd sources, use:
+##     target_SOURCES = $(tincd_common_SOURCES) unittest/test_name.c
+##     target_LDFLAGS = $(tincd_LDFLAGS)
+##     target_LDADD   = $(tincd_test_LDADD)
+## To link against tinc, replace `tincd` with `tinc` in the directives above.
+## Paths below are relative to src/
+
+TESTS += \
+	test_subnet \
+	test_splay_tree
+
+test_subnet_SOURCES = $(tincd_common_SOURCES) unittest/test_subnet.c
```

Comment:
The sources must also list the header files that are used, otherwise those are not copied into the tarball when running `make dist`.